index.html

```diff
@@ -52,5 +58,126 @@ <h1>Hey there. I'm Emmanuel</h1>
         </li>
         </ul>
     </section>
-</body>
-</html>
\ No newline at end of file
+
+    <section class="myprojects">
+      <h2 class="project-title">My Recent Works</h2>
+      <article class="myprojects project">
+        <div class="description">
+          <h3>Multi-Post Stories</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required 
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+        <ul class="skills">
+          <li class="css">css</li>
+          <li class="html">html</li>
+          <li class="boot">Bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+          <button type="button" class="btn">See Project</button>
```

Comment:
- [ ] kindly add a link place holder `(a tag)` so it will make it easier to navigate

Reply:
Hi @tufoinnkuo10 
Thank you for the review. I have a question, though. 
Did you mean I add the (a tag) on all the "See Project" butons?